pytition/petition/templates/petition/index.html

```diff
@@ -1,11 +1,55 @@
 {% extends 'petition/base.html' %}
 {% load static %}
 {% load i18n %}
-{% block container_type %}container{% endblock container_type %}
 
 {% block login_next %}{% url "user_dashboard" %}{% endblock login_next %}
+
+{% block extracss %}<link href="{% static "css/home.css" %}" rel="stylesheet">{% endblock extracss %}
 {% block main_grid %}
+  <section class="jumbotron text-center">
+    <div class="container">
+      <h1 class="jumbotron-heading">Bienvenue sur Pytition</h1>
+      <p class="lead text-muted">Mobilisez-vous pour faire changer les choses !</p>
```

Comment:
Could you please use the internationalization primitive?
`{% trans "text" %}` for small texts, or 
```
{% blocktrans %}
bla
bla
{% endblocktrans %}
```

for long texts.

Reply:
Done 👍